sdk/python/src/freewayai/llm_system.py

```diff
@@ -1,6 +1,7 @@
-from .prompt_template_format import read_and_format_prompt
 from .prompt_formatted import Prompt_Formatted
 from .database_query_template_format import read_and_format_query
+from .utilities import _read_prompt_from_system_and_id
```

Comment:
Calling out here that it's weird to import a function that you've put an underscore in front of. You probably want to rename the function if you want to think of it as public.